.github/scripts/rust/setup.py

```diff
@@ -24,10 +24,11 @@
 # rust-toolchain.toml
 TOOLCHAINS = {
     "packages/libs/error-stack": ["1.63", "beta-2022-09-25"],
+    "packages/libs/deer": ["1.63", "beta-2022-09-25"]
 }
 
 # Try and publish these crates when their version is changed in Cargo.toml
-PUBLISH_PATTERNS = ["packages/libs/error-stack**"]
+PUBLISH_PATTERNS = ["packages/libs/error-stack**", "packages/libs/deer**"]
```

Comment:
We *don't* want to publish I think?

Reply:
Yep, this also led to a failure of the CI and has since been removed.

Reply:
removed in [4ba1939](https://github.com/hashintel/hash/pull/1163/commits/4ba193947e063f825a9765b8503fcd001429d698)